adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -78,6 +79,9 @@ class CircuitPlaygroundBase:  # pylint: disable=too-many-public-methods
 
     _audio_out = None
 
+    # Circuit Playground specific board names as found in os.uname().machine
+    _CP_TYPES = ("Bluefruit", "Express")
```

Comment:
@dhalbert would you think doing this consts make more sense?

Reply:
I suggested consts as a possibility but @kattni preferred strings. We need the strings anyway to search for them in the `os.uname()` field to distinguish the boards.

We could use string consts.